src/components/transactions/TransactionRow.vue

```diff
@@ -74,6 +77,10 @@
           The receiver address {{ transaction.receiverAddress }} is not your currently connected
           account {{ sharedState.accountAddress }}
         </p>
+        <p v-if="toNetwork.isRsk" class="alert alert-warning" role="alert">
+          Binance is not taking deposits sent by a smart contract for RSK network, they only accept
```

Comment:
is this specific for Binance? maybe we can extract this stuff from a variable idk

Reply:
actually this is specific for RSK, that why it has the  v-if="toNetwork.isRsk"